apps/api/src/types/index.ts

```diff
@@ -737,6 +737,14 @@ export const UpdateAgentProfileBodySchema = z
       .min(1, { message: "Description must be at least 1 character" })
       .optional(),
     imageUrl: z.url("Invalid image URL format").optional(),
+    metadata: z
```

Comment:
this change isn't needed. if you remove it, a user is able to successfully update the agent skills w/o issue. (i tested this locally.)